### ISSUE TITLE
F5tts stopped working with version 3.2

### ISSUE THREAD

**Issue by edbartz** [JarodMica/audiobook_maker#102]
I installed version 3.2, but when I try to run f5tts voices I get:
  state_dict = torch.load(model_path, map_location="cpu")

vocab :  C:\Users\xxx.xxx\Desktop\TTS\audiobook_maker\runtime\Lib\site-packages\f5_tts\infer\examples\vocab.txt
tokenizer :  custom
model :  engines\f5tts\models\models--SWivid--F5-TTS\snapshots\4dcc16f297f2ff98a17b3726b16f5de5a5e45672\F5TTS_Base\model_1200000.safetensors

Error calling Python override of QThread::run(): Traceback (most recent call last):
  File "C:\Users\xxx.xxx\Desktop\TTS\audiobook_maker\src\controller.py", line 109, in run
    new_audio_temp_path = self.model.generate_audio_proxy(self.selected_sentence, self.combined_parameters, s2s_validated)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\xxx.xxx\Desktop\TTS\audiobook_maker\src\model.py", line 310, in generate_audio_proxy
    success = tts_engines.generate_audio(self.tts_engine, sentence, voice_parameters, tts_engine_name, audio_path)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\xxx.xxx\Desktop\TTS\audiobook_maker\src\tts_engines.py", line 39, in generate_audio
    return generate_with_f5tts(tts_engine, sentence, voice_parameters, audio_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\xxx.xxx\Desktop\TTS\audiobook_maker\src\tts_engines.py", line 145, in generate_with_f5tts
    speed = round(voice_parameters.get("f5tts_speed") / speed_step, 2)
                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~

Now I noticed the last lines refer to parameter "f5tts_speed", but when I look at the config file, there is no "f5tts_speed".    If I print these variables,  I get voice_parameters.get("f5tts_speed") = NONE and speed_step = 100.
Ed

**Comment by edbartz** [comment]
F5tts is now working correctly.  I like the speed adjustment, as I felt the voices were a little fast.

And Yes, the install went perfectly.  I had to do nothing but run the install bat file for styletts.  

You know I spent two days trying to fix that combo box problem in 3.1.   Never could figure out how to fix it, though I suspected the problem might be where you fixed it, but I could not tell.   It makes me a little discouraged for adding my word replacer back in,
but we shall see.  I also want to see about some of my changes to the text file read in.  Removing characters, etc.

Thanks again for all your hard work.   You managed to teach this old dog a few things  :-).  The I still have a lot to learn.

Ed
aka Physengineer

**Comment by JarodMica** [comment]
> F5tts is now working correctly. I like the speed adjustment, as I felt the voices were a little fast.
> 
> And Yes, the install went perfectly. I had to do nothing but run the install bat file for styletts.
> 
> You know I spent two days trying to fix that combo box problem in 3.1. Never could figure out how to fix it, though I suspected the problem might be where you fixed it, but I could not tell. It makes me a little discouraged for adding my word replacer back in, but we shall see. I also want to see about some of my changes to the text file read in. Removing characters, etc.
> 
> Thanks again for all your hard work. You managed to teach this old dog a few things :-). The I still have a lot to learn.
> 
> Ed aka Physengineer

I'm glad to provide any bit of motivation that I can :)!  If you could remind me what the word replacer was and how the logic worked, I can think of a way to add that to my list of improvements. Is it similar to https://github.com/JarodMica/audiobook_maker/issues/94?

If not, feel free to add an issue and I'll add the enhancement label to it
